lua/dotfyle_metadata/extras/lsp.lua

```diff
@@ -7,7 +7,7 @@ return function()
 		return {}
 	end
 
-	local servers = lspconfig.util.available_servers()
+	local servers = lspconfig.util._available_servers()
```

Comment:
Hey thanks for the PR!

Would you be able to keep the previous function call and do a check for the new function?

Since this change is very recent (2 days ago from today) not many users would've updated the plugin (like myself).

I think it's best to still keep the old function for some time before we rename it completely to the new `_available_servers`.

I'm thinking like the following conditional pseudocode:
- If `_available_servers` is not nil then call it
- Otherwise, call `available_servers`

Reply:
I'm getting an error where `_available_servers` is not available, If I'm in the older version of lspconfig.

<img width="778" alt="image" src="https://github.com/user-attachments/assets/6e9ad358-c048-4a8d-8937-bc6b03271770" />

I think it's best to compare if the function itself is defined instead of calling the function.

So check `lspconfig.util._available_servers ~= nil` (without the braces `()`)

Then `servers = lspconfig.util._available_servers()` (call the function) otherwise `servers = lspconfig.util.available_servers()`